pkg/netpol/eval/resources.go

```diff
@@ -285,20 +327,25 @@ func (pe *PolicyEngine) HasPodPeers() bool {
 }
 
 // createPodOwnersMap creates map from workload str to workload peer object
```

Comment:
```suggestion
// createPodOwnersMap creates map from workload str to workload peer object
// returns error if there are two pods of same owner but different set of labels, since cannot map inconsistent pods to a workload
```